core/src/com/unciv/ui/mapeditor/TileEditorOptionsTable.kt

```diff
@@ -163,12 +163,12 @@ class TileEditorOptionsTable(val mapEditorScreen: MapEditorScreen): Table(Camera
                 val improvementName = "StartingLocation " + nation.name
                 tileAction = {
                     it.improvement = improvementName
-                    for (tileGroup in mapEditorScreen.mapHolder.tileGroups.values) {
-                        val tile = tileGroup.tileInfo
+                    for (tileGroupEntry in mapEditorScreen.mapHolder.tileGroups) {
```

Comment:
`for((tile, tileGroups) in mapEditorScreen.mapHolder.tileGroups)`

Reply:
that's possible? :O nice
done